GLOSSARY.md

```diff
@@ -15,8 +15,8 @@
 - `Location` / `Range`
 - `Freshness` - When a literal type is first created and not expanded by hitting a mutable location, see [Widening
   and Narrowing in TypeScript][wnn].
-- `Symbol` - The binder creates
-- `Transient Symbol` -
+- `Symbol` - An object that tracks all the places a variable or type is declared
+- `Transient Symbol` - A symbol created in the checker
```

Comment:
I know you mention the binder creating symbols too later, but maybe it would be more clear to complete this with something like "A symbol created in the checker, as opposed to in the binder"?